### PR TITLE
Clean up functional test for 'create case' endpoint

### PR DESCRIPTION
**Pull request by rkondratowicz**
just some minor changes in preparation for adding a new test for new search type

### CODE REVIEW

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/CreateCaseTest.java [hunk 1]
```diff
@@ -162,26 +145,25 @@ private AboutToStartOrSubmitCallbackResponse invokeCallbackEndpoint(
             .header(SyntheticHeaders.SYNTHETIC_TEST_SOURCE, "Bulk Scan Orchestrator Functional test")
             .header(AUTHORIZATION, ccdAuthenticator.getUserToken())
             .header(CcdCallbackController.USER_ID, ccdAuthenticator.getUserDetails().getId())
-            .body(callbackRequest)
+            .body(
+                CallbackRequest
+                    .builder()
+                    .eventId(EventIds.CREATE_NEW_CASE)
+                    .caseDetails(exceptionRecord.toBuilder().build())
```
Comment (doncem) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/CreateCaseTest.java:152]: why do we need `toBuilder().build()`?

Comment (rkondratowicz) [reply src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/CreateCaseTest.java:152]: not sure. the (now gone) variable had a mysterious name `exceptionRecordWithSearchFields`

Comment (doncem) [reply src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/CreateCaseTest.java:152]: just checked case details. the only difference is just creating brand new object - no hidden changes

Comment (rkondratowicz) [reply src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/CreateCaseTest.java]: ok, removing

**Pull request merged**